### PR TITLE
Csanád - address chapter 14 tech review and editorial comments

### PR DESCRIPTION
**Pull request by Xronophobe**

### CODE REVIEW

File: chapter_14_simple_form.asciidoc [hunk 1]
```diff
@@ -102,7 +102,6 @@ It's already pretty close to what we have in _base.html_.  We're missing
 the placeholder attribute and the Bootstrap CSS classes.  Let's make our
 unit test into a test for that:
 
-// JAN: Would be great to have imports inside the code snippets below
```
Comment (Xronophobe) [inline chapter_14_simple_form.asciidoc:105]: We have, in the first code snippet above, in this section. Have we missed any?

File: chapter_14_simple_form.asciidoc [hunk 2]
```diff
@@ -165,6 +164,7 @@ And then:
 ----
 ====
 // RITA: Is this note missing "for some help" at the end?
+// CSANAD: Not sure what you mean?
```
Comment (Xronophobe) [inline chapter_14_simple_form.asciidoc:167]: @ritaf-ORM could you please clarify?

File: chapter_14_simple_form.asciidoc [hunk 3]
```diff
@@ -719,7 +718,6 @@ class NewListTest(TestCase):
 ----
 ====
 
-// JAN: Haven't we removed NewListTest in previous chapter and placed all tests to ListViewTest?
```
Comment (Xronophobe) [inline chapter_14_simple_form.asciidoc:722]: No, it was `NewItemTest` that we retired.

File: chapter_14_simple_form.asciidoc [hunk 4]
```diff
@@ -188,6 +188,19 @@ You might use the interactive console, or write some exploratory code
...
+To use it, simply put a `breakpoint()` in your code where you
+would like it to pause and enable you to look around.footnote:[Before Python 3.7
+you would invoke the debugger by placing `import pdb; pdb.set_trace()` in your
+code.]
+Once the execution is halted, you can print variables, results of expressions,
+list the source code lines surrounding where your program is currently at,
+step your code through each line and even more! Type `h` and press enter to
+list all your options and
+https://docs.python.org/3/library/pdb.html#debugger-commands[have a look at the
+documentation].
```
Comment (Xronophobe) [inline chapter_14_simple_form.asciidoc:202]: re: https://github.com/hjwp/Book-TDD-Web-Dev-Python/pull/297

File: chapter_14_simple_form.asciidoc (hunk 2 above)
Comment (ritaf-ORM) [reply chapter_14_simple_form.asciidoc:167]: In the previous edition of the book, this note box ended with "Check out django-crispy-forms
and django-floppyforms for some help." You removed the mention of django-floppyforms, which is fine. I'm asking if you accidentally removed "for some help" to complete the sentence.

File: chapter_14_simple_form.asciidoc (hunk 4 above)
Comment (hjwp) [reply chapter_14_simple_form.asciidoc:202]: i think if we start talking about pdb, then we need a much more detailed explanation.

Comment (hjwp) [reply chapter_14_simple_form.asciidoc:202]: so i'd rather just not mention it tbh.  i think i already put it in a footnote somewhere, i'd rather not get into it

Comment (Xronophobe) [reply chapter_14_simple_form.asciidoc:202]: ACK, removing it soon

**Pull request merged**